cmd/kubelet/app/options/options.go

```diff
@@ -23,7 +23,6 @@ import (
 
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/apiserver/pkg/util/flag"
```

Comment:
Thanks for pointing out this, I also have thought about this before submit this PR. But after go through other pkgs, I find that this style is necessary only when there are package conflicts. I think it is briefer to directly import without alias when on package conflicts.